Lesson8/src/com/handlers/HandlerLesson2.java

```diff
@@ -17,8 +17,7 @@ public void handleSomeString(String string) throws IOException {
         String[] formattedData = TextFormatter.getSentences(string);
         TmsWriter anotherWriter = new TmsFileWriter(OUTPUT_PATH);
         for (String sentence: formattedData){
-            System.out.println(TextFormatter.getNumbWords(sentence));
-            if (TextFormatter.checkPalindrome(sentence)){
+            if (TextFormatter.checkPalindrome(sentence) || (TextFormatter.getNumbWords(sentence) >= 3 && TextFormatter.getNumbWords(sentence) <= 5)){
```

Comment:
I would recommend writing result of TextFormatter.getNumbWords to a variable to avoid calculating the same value twice

Reply:
done